src/GeositeFramework/Models/Geosite.cs

```diff
@@ -142,8 +142,23 @@ private static List<JsonData> GetPluginConfigurationData(IEnumerable<string> plu
             jsonObj.Add("pluginFolderNames", new JArray(PluginFolderNames.ToArray()));
 
             // Set public properties needed for View rendering
-            TitleMain = ExtractLinkFromJson(jsonObj["titleMain"]);
-            TitleDetail = ExtractLinkFromJson(jsonObj["titleDetail"]);
+            if (SinglePluginMode)
+            {
+                TitleMain = new Link
+                {
+                    Text = (string)jsonObj["singlePluginMode"]["title"],
```

Comment:
You're right. This probably isn't needed. I was originally thinking they would want to switch it on and off depending on the deployment, but that probably isn't the case. I'll revise.